BZ2034184: Note that Ignition configs should be used shortly after install (#40238)
openshift/openshift-docs: modules/installation-aws-user-infra-installation.adoc

+
[IMPORTANT]
====

> * The Ignition config files that the installation program generates contain certificates that expire after 24 hours, which are then renewed at that time. If the cluster is shut down before renewing the certificates and the cluster is later restarted after the 24 hours have elapsed, the cluster automatically recovers the expired certificates. The exception is that you must manually approve the pending `node-bootstrapper` certificate signing requests (CSRs) to recover kubelet certificates. See the documentation for _Recovering from expired control plane certificates_ for more information.

Reviewer: ```suggestion
* The Ignition config files that the installation program generates contain certificates that expire after 24 hours. The certificates will automatically be renewed by the cluster some time, up to a few hours, before the certificates expire. If the cluster is shut down before renewing the certificates and the cluster is later restarted after the 24 hours have elapsed, the cluster automatically recovers the expired certificates. The exception is that you must manually approve the pending `node-bootstrapper` certificate signing requests (CSRs) to recover kubelet certificates. See the documentation for _Recovering from expired control plane certificates_ for more information.
```

Reviewer 2: maybe we could also clarify the renewed time more accurately like the following mentioned in https://access.redhat.com/solutions/4271712
```
The rotation threshold of the OpenShift (kubernetes) certificate manager is determined based on the life of the certificate. Certificates are rotated between the 70% and 90% (target 80%) validity period.
For these 24 hour certificates, the rotation will happen between 16 and 22 hours after cluster installation.
```

Author: It was suggested that we recommend 12 hours, so that has been added as part of the wording.

> * It is recommended that Ignition config files are used shortly after they are generated because certificate rotation can occur at anytime, even before the 24-hour certificate expiration.

Reviewer: ```suggestion
* It is recommended that Ignition config files are used shortly after they are generated to avoid issues where the cluster may attempt to renew certificates during cluster installation. The cluster may renew certificates hours before the 24-hour expiration.
```
